base/operators.jl

```diff
@@ -1149,40 +1149,96 @@ julia> filter(!isletter, str)
 !(f::ComposedFunction{typeof(!)}) = f.inner #allows !!f === f
 
 """
-    Fix1(f, x)
+    Fix{n}(f, x)
+    Fix{kw}(f, x)
+    Fix(f; [kw]=x)
 
-A type representing a partially-applied version of the two-argument function
-`f`, with the first argument fixed to the value "x". In other words,
-`Fix1(f, x)` behaves similarly to `y->f(x, y)`.
+A type representing a partially-applied version of a function `f`, with the argument
+"x" fixed at argument `n::Int` or keyword `kw::Symbol`.
+In other words, `Fix{3}(f, x)` behaves similarly to
+`(y1, y2, y3) -> f(y1, y2, x, y3)` for the 4-argument function `f`.
 
-See also [`Fix2`](@ref Base.Fix2).
+You may also use this to fix keyword arguments. For example, `Fix(g; a=2)` behaves
+similarly to `x -> g(x; a=2)` for a function `g` with one argument and one keyword argument.
+You can also write this as `Fix{:a}(g, 2)`.
 """
-struct Fix1{F,T} <: Function
+struct Fix{N,F,T} <: Function
     f::F
     x::T
 
-    Fix1(f::F, x) where {F} = new{F,_stable_typeof(x)}(f, x)
-    Fix1(f::Type{F}, x) where {F} = new{Type{F},_stable_typeof(x)}(f, x)
+    function Fix{N}(f::Union{F,Type{F}}, x) where {N,F}
```

Comment:
```suggestion
    function Fix{N}(f::F, x) where {N,F}
```
The reason for this split definition in the original was to replicate `_stable_typeof`. Now that we're using that function explicitly, there's no need for this convoluted type signature.